app/src/main/java/uk/gov/onelogin/ui/home/HomeScreen.kt

```diff
@@ -27,11 +27,12 @@ fun HomeScreen(
 ) {
     viewModel.saveTokens(LocalContext.current as FragmentActivity)
     val tokens = viewModel.getTokens()
+    val email = viewModel.getEmail() ?: ""
```

Comment:
See below, think this should be `LiveData`

Reply:
same 1d72c76af5b623681ee7d7cd31d46f289a480372